config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      get 'messages/index'
```

Comment:
- [ ] Would you please specify the route by directing it to a controller's method? Ex: `get 'messages/index' to: 'messages#index'`